python/lsst/obs/base/gen2to3/rootRepoConverter.py

```diff
@@ -119,16 +120,19 @@ def runRawIngest(self, pool=None):
             dataRefs = self.butler2.subset(self.task.config.rawDatasetType)
         dataPaths = getDataPaths(dataRefs)
         self.task.log.info("Ingesting raws from root %s into run %s.", self.root, self.task.raws.butler.run)
-        self._rawRefs.extend(self.task.raws.run(dataPaths, pool=pool))
+        if not self.task.dry_run:
+            self._rawRefs.extend(self.task.raws.run(dataPaths, pool=pool))
         self._chain = [self.task.raws.butler.run]
 
     def runDefineVisits(self, pool=None):
         if self.task.defineVisits is None:
+            self.task.log.info(f"Skipping visit definition for {self.root}.")
             return
         dimensions = DimensionGraph(self.task.universe, names=["exposure"])
         exposureDataIds = set(ref.dataId.subset(dimensions) for ref in self._rawRefs)
         self.task.log.info("Defining visits from exposures.")
-        self.task.defineVisits.run(exposureDataIds, pool=pool)
+        if not self.task.dry_run:
+            self.task.defineVisits.run(exposureDataIds, pool=pool)
```

Comment:
As above may a log message saying that N exposureDataIds would have had visits defined?